contrib/signet/getcoins.py

```diff
@@ -8,6 +8,7 @@
 import requests
 import subprocess
 import sys
+import xml.etree.ElementTree
```

Comment:
```suggestion
import defusedxml.ElementTree
```


Sorry for the late review. I was still going through everything in the code and testing things. Found that `xml.etree.ElementTree.fromstring` is known to be vulnerable to XML attacks. Maybe this can be changed in a follow up PR.

Reply:
*If* there is a specific problem with `xml.etree` it needs to be addressed. But without introducing a third-party dependency, please. Adding `defusedxml` is not acceptable here.

Reply:
I will try fixing it without adding dependency.

Details: https://deepsource.io/gh/prayank23/bitcoin/run/61a5e60f-9376-4fd4-8a65-3b76190a56af/python/BAN-B314

I found the issue using deepsource, it's really helpful for python but they don't support C++ yet. Will be helpful if some developers could upvote/share this: https://deepsource.canny.io/analyzer-requests/p/c-support-2